### ISSUE TITLE
Feature: Implement X so users can do Y (REPLACE THIS)

### ISSUE THREAD

**Issue by kylehovey** [bitgapp/eqMac#624]
**Disclaimer:** 
Please go through [existing feature requests](https://github.com/bitgapp/eqMac/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+feature%22) to avoid creating duplicates. If you found a similar issue please post a comment there. If you are uncertain your issue is related to others then create the issue. Obvious ignorance of this rule might get you blocked from posting new issues. Please tick the box below to show that you have followed the guideline.
- [x] I have checked for a similar issue and sure it hasn't been reported before.

**Description:**
I would love to be able to switch between a minimum phase EQ and linear phase. The minimum phase EQ (which I think is what is currently there) affects the phase information in the tracks which can affect imaging negatively for a lot of songs. Linear phase EQ has the downside of pre-ringing, but I think the better imaging response would be worth having as an option. I don't know how hard this would be to add, but if this was added I would absolutely pay for the full version.

**User flow & interface:**
Adding a small dropdown for what type of filter is used for EQ could go either in the Advanced parametric EQ or in the Expert visual EQ.

![image](https://user-images.githubusercontent.com/7339800/149637220-c8a5c75b-730b-41ab-993d-8030d8ccc60b.png)

**Comment by andresbrocco** [comment]
Up on this feature!

**Comment by Elliottmeija** [comment]
`The minimum phase EQ (which I think is what is currently there) affects the phase information in the tracks which can affect imaging negatively for a lot of songs.`

Audio engineer here! While this is technically true, what is likely more noticable to most is the spacial properties of each band's audio, rather than the phase smearing causing it as you describe. I'm sure you have mastered a few songs yourself but ultimately we forget our ears are trained.

This slight difference doesn't account for pre-ringing and CPU load, which consumers will notice much, much more and likely hate.

I personally wouldn't use linear phase live.

**Comment by kylehovey** [comment]
@Elliottmeija good point, and the pre-ringing can produce some unwanted artifacts as well. I recently tried out SoundID Reference, which provides a minimum phase, linear, and a mixed EQ that compromises between the two. The app lets you chart the phase response, and the middle filter is actually not that bad in terms of phase distortion while sounding better to my ears in terms of spatialization and imaging for many tracks over the minimum phase EQ.

I realize that sort of compromise is probably very difficult to get right, but it may be a good option here if available developer bandwidth is available.

**Comment by Elliottmeija** [comment]
@kylehovey that's an awesome feature, and I'm impressed that they got it right. Time for some red warning text for sure if this feature is implemented, but the need for it could be mitigated if memory is managed properly.
